tests/unit/amo/pages/TestCollection.js

```diff
@@ -1392,6 +1371,67 @@ describe(__filename, () => {
       }),
     );
   });
+  it('sends a server redirect when username is not lower case', () => {
+    const clientApp = CLIENT_APP_FIREFOX;
+    const lang = 'fr';
+
+    const { store } = dispatchClientMetadata({ clientApp, lang });
+    const fakeDispatch = sinon.spy(store, 'dispatch');
+
+    _loadCurrentCollection({ store });
+
+    fakeDispatch.resetHistory();
+
+    const collection = getCurrentCollection(store.getState().collections);
+
+    const params = {
+      slug: collection.slug,
+      username: collection.authorUsername.toUpperCase(),
```

Comment:
Nit: As with the nit I filed above, I think we should set a known value for the collection `authorUsername` rather than assuming it's not already upper case.